ansible/roles/providers/tasks/vmware.yml

```diff
@@ -35,7 +35,7 @@
     packages:
       - cloud-init
       - cloud-utils-growpart
-      - python2-pip
+      - "python{{ '3' if ansible_distribution_major_version == '9' else '2'}}-pip"
```

Comment:
So we only want python 3 for rhel or rocky 9?

Reply:
Rocky linux claims to be compatible with RHEL from version 9 and newer. rocky 9.x uses python3-* packages. We dont support RHEL 9.X now, but we will need python3-* for RHEL too. condition `ansible_distribution_major_version == '9'` will cover both usecaes. however, now I realize that it would be better set it for 9.x and above.

I can change the condition to `(ansible_distribution_major_version | int)  >= 9`.  wdyt ?

Reply:
that makes sense to me. 

my comment was thinking ahead to the point where we do support rhel 9.